consensus/ibft/signer/extra.go

```diff
@@ -37,6 +42,35 @@ type Seals interface {
 	UnmarshalRLPFrom(*fastrlp.Parser, *fastrlp.Value) error
 }
 
+// parseRoundNumber parses RLP-encoded bytes into round
+// FYI, Extra has 8 bytes space for round in order to distinguish between null and 0
```

Comment:
If I'm correct this comment is now redundant?

```suggestion
```

Reply:
No

Reply:
@Stefan-Ethernal

Reply:
I was referring that round is not nil and in that case it is 8 bytes length or if it is `nil` it's length is 0. With having that on mind I'm not sure if comment was trying to outline the same thing, because it is a bit confusing to me.

Reply:
Round in IBFT Extra can be nil in order to support backward compatibility

Reply:
In case of that Extra has Round, bytes data would have non-zero size always.

Reply:
Fixed comment 63f4cc01137fd2e71b99d270de89bd232a6b05cd. Is that fine with you @Stefan-Ethernal ?

Reply:
> Round in IBFT Extra can be nil in order to support backward compatibility

Yes aware of it and it is completely fine 👍 . However I was referring that comment is somewhat imprecise, since we would have either 8 bytes or 0 (in case round is `nil`). 

Don't mind keeping it, but just wanted to make this clear.

Reply:
> Fixed comment [63f4cc0](https://github.com/0xPolygon/polygon-edge/commit/63f4cc01137fd2e71b99d270de89bd232a6b05cd). Is that fine with you @Stefan-Ethernal ?

Yep this is better. Tnx 🙏

Reply:
@Stefan-Ethernal ping